com.unity.multiplayer.mlapi/Tests/Runtime/MultiInstance/BaseMultiInstanceTest.cs

```diff
@@ -0,0 +1,92 @@
+using System;
+using System.Collections;
+using NUnit.Framework;
+using UnityEngine;
+
+namespace MLAPI.RuntimeTests
+{
+    public class BaseMultiInstanceTest
+    {
+        private int m_OriginalTargetFrameRate;
+
+        protected GameObject m_PlayerPrefab;
+
+        protected NetworkManager m_ServerNetworkManager;
+        protected NetworkManager[] m_ClientNetworkManagers;
+
+        public virtual void Setup()
+        {
+            // Just always track the current target frame rate (will be re-applied upon TearDown)
+            m_OriginalTargetFrameRate = Application.targetFrameRate;
+
+            // Since we use frame count as a metric, we need to assure it runs at a "common update rate"
+            // between platforms (i.e. Ubuntu seems to run at much higher FPS when set to -1)
+            if (Application.targetFrameRate < 0 || Application.targetFrameRate > 120)
+            {
+                Application.targetFrameRate = 120;
+            }
```

Comment:
this trick will no longer be needed after #893

Reply:
@MFatihMAR given #893 is merged already would you recommend we remove this then?

Reply:
yes.